scripts/azure-linux_mac.yml

```diff
@@ -156,13 +156,10 @@ steps:
     fi
 
     # GCS unit tests are temporarily unsupported on CI.
-    if [[ "$TILEDB_GCS" == "ON" ]]; then
-      ./tiledb/test/tiledb_unit --durations=yes --vfs native ~[gcs]	
-    else	
+    if [[ "$TILEDB_GCS" != "ON" ]]; then
```

Comment:
@ihnorton I'm having to make a few changes to the CI script to avoid executing anything (tests, examples) for the GCS build because they all fail when trying to connect to GCS (because we don't have an emulator and we're not configuring them to run on production). 
As a reminder, I intend to hook up an emulator or something later, but we've deferred it for now.

I'll let you know once this passes CI so you can re-review the changes I'm making here.